### DISCUSSION TITLE
starting script

### DISCUSSION THREAD

**Discussion by jakubTro26** [php-webdriver/php-webdriver#1017] [General]
I'm running php script.php and nothing happens.
No output is visible, but I'm echoing.

There must be error in code but my file is:

`<?php  

use Facebook\WebDriver\Remote\RemoteWebDriver;
use Facebook\WebDriver\Remote\DesiredCapabilities;

$serverUrl = 'http://localhost:4444'; // if you don't start chromedriver with "--port=4444" as above, default port will be 9515
$driver = RemoteWebDriver::create($serverUrl, DesiredCapabilities::chrome());

$driver->get('https://en.wikipedia.org/wiki/Selenium_(software)');

echo 'kuba';`

everything works when i leave echo alone

**Comment by ernstki** [comment]
Wish I could've helped, but it's been years ago already. Do you remember how you finally solved this?
